src/pages/Profile.vue

```diff
@@ -51,85 +51,107 @@
                 >{{ $t('actions.view_id_badge') }}
               </base-button>
             </div>
-            <div class="user-form p-8">
+            <div class="user-form p-10 sm:p-8">
               <form ref="form" @submit.prevent="handleSubmit">
                 <div class="user-details">
                   <div class="flex pb-4">
-                    <base-input
-                      class="mr-2 w-1/2"
-                      size="large"
-                      :value="currentUser.first_name"
-                      :placeholder="$t('profileUser.first_name_placeholder')"
-                      required
-                      @input="
-                        (value) => {
-                          updateUser(value, 'first_name');
-                        }
-                      "
-                    />
-                    <base-input
-                      class="w-1/2"
-                      size="large"
-                      :value="currentUser.mobile"
-                      :placeholder="$t('profileUser.mobile_placeholder')"
-                      required
-                      :validator="validatePhoneNumber"
-                      @input="
-                        (value) => {
-                          updateUser(value, 'mobile');
-                        }
-                      "
-                    />
+                    <div class="form-field mr-2">
+                      <label for="first_name">First Name</label>
```

Comment:
```suggestion
                      <label for="first_name">$t('profileUser.first_name_placeholder')</label>
```

Don't forget localizations :wink: 
If you add some new text, be sure to add `~~` so that Aaron can add the localizations later. Ex: `$t('~~Hello world')`

Reply:
done